src/languages/cs.js

```diff
@@ -1,25 +1,37 @@
+  }
+
```

Comment:
There's one big unsolvable problem with this: as soon as a language gets into the "common" set (C# is one of them) we really don't want to blow up its size because "common" is what comprises our CDN build. So we simply can't have an exhaustive list of builtins from a big standard library.